combat/attack.cpp

```diff
@@ -194,7 +200,7 @@ bool Creature::canAttack(const std::shared_ptr<Creature>& target, bool stealing)
         return(false);
     }
 
-    if(isCt()) {
+    if(isCt() && flagIsSet(P_SITTING)) {
```

Comment:
Should this be an OR condition? I remember you mentioning not having to stand before attacking anymore, but this would only affect CTs/DMs

Reply:
I guess technically that might not even need to be there. I don't know why i thought it needed a separate check just for CTs. It's called below that on line 234 anyway. I think it had to do with some kind of output OCD shit i was trying to eliminate. I'll see if i can just get rid of it.

Reply:
Switched this so stand() checks for P_DM_INVIS and removed the redundant P_SITTING check for CT+